Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -944,3 +944,14 @@ end OrderMonoidWithZeroHom
 def OrderMonoidIso.unitsWithZero {α : Type*} [Group α] [Preorder α] : (WithZero α)ˣ ≃*o α where
   toMulEquiv := WithZero.unitsWithZeroEquiv
   map_le_map_iff' {a b} := by simp [WithZero.unitsWithZeroEquiv]
+
+/-- A version of `Equiv.optionCongr` for `WithZero` on `OrderMonoidIso`. -/
+@[simps!]
+def OrderMonoidIso.withZero {G H : Type*}
+    [CommGroup G] [PartialOrder G] [CommGroup H] [PartialOrder H] :
```

Comment:
Not at my computer now, but do you need commutativity?

Reply:
No, will remove.